config/betamocks/services_config.yml

```diff
@@ -58,11 +58,11 @@
   :base_uri: <%= URI(Settings.mvi.url).host %>
   :endpoints:
   - :method: :post
-    :path: "/psim_webservice/IdMWebService"
+    :path: <%= URI(Settings.mvi.url).path %>
     :file_path: "mvi/profile"
     :cache_multiple_responses:
       :uid_location: body
-      :uid_locator: 'root="2.16.840.1.113883.4.1" extension="(\d{9})"'
+      :uid_locator: '(?:root="2.16.840.1.113883.4.1" )?extension="(\d{9})"(?: root="2.16.840.1.113883.4.1")?'
```

Comment:
AFAI knew, this already worked for local dev environments.  I assume you're making this change for `dev` or `staging`.  Can you confirm that this still works for local dev?

Reply:
Yes, we are seeing that Ox can generate the MVI request with these attributes in either order. This regex accepts both variants.